.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,58 @@
+<!--
+    Hello! Thanks for submitting a pull request to MusicBrainz Server
+    We appreciate your time and interest in helping our project!
+
+    Use this template to help us review your change. Not everything is required,
+    depending on your change. Keep or delete what is relevant for your change.
+    Remember that it helps us review if you give more helpful info for us to
+    understand your change.
+
+    Ensure that you’ve read through and followed the Contributing Guidelines, in
+    https://github.com/metabrainz/musicbrainz-server/blob/master/CONTRIBUTING.md
+-->
+
+# Summary
+<!--
+    Just give us a overview of your change, don’t go into technical details yet.
+-->
+
+:beginner: Describe this change in 1-2 sentences
+
+
+# Problem
+<!--
+    Anything that helps us understand why you are making this change goes here.
+    What problem are you trying to fix? What does this change address?
+-->
+
+:beginner: Reference related tickets with MBS-XXX at least.
+
+
+# Solution
+<!--
+    Talk about technical details, considerations, or other interesting points.
+-->
+
+:beginner: If you have a lot to say, be more detailed in this section.
+
+
+# Readiness
+<!--
+    The tasks you have to do to get your change ready for review. Use this if
+    you draft a pull request. Mark done tasks with an [x] as you progress. See
+    https://github.blog/2019-02-14-introducing-draft-pull-requests/
+-->
+
+* [x] Read the [Contributing Guidelines](https://github.com/metabrainz/musicbrainz-server/blob/master/CONTRIBUTING.md)
```

Comment:
One thing we saw in listenbrainz with this is that if you add checkboxes, github will add "completed 1/6 tasks" on the PR list page, which gets annoying. We replaced it with (x) instead of []

Reply:
Yes, that is why I did not keep the issue type selector (from the Picard PR template) and use it as a to-do list (as intended), see commit https://github.com/metabrainz/musicbrainz-server/pull/1345/commits/2490c7eb11616cdeb207752c43cbe01f6d03b517 message.

Reply:
oh, I see, so these actually _are_ tasks that you want captured. Fine 👍

Reply:
Yup, I checked the first item as an example and because it has usually been done already except maybe for the very first PR. The next item isn’t checked as an example too and to make sure it will actually be done (removing that item or section is a way to do it.)